Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12.19.0-alpine
```

Comment:
ignore infrastructure files

Reply:
Please focus changes under `server` folder. Rest are a template copy of what was in wallet-api.